src/Form/matching-config/matching-config-source.js

```diff
@@ -416,7 +416,7 @@ const matchingConfiguration = {
                 },
                 resetPasswordLink: {
                     match:
-                        "(forgot(ten)?|reset|don't remember) (your )?password|password forgotten" +
+                        "(forgot(ten)?|reset|don't remember)[-\\s]?(your )?password|password forgotten" +
```

Comment:
This allows `forgot-password` to be matched, which is a valid case. Note that this also allows `forgotPassword` to be matched now, creating the test results to change (see comment below).